lib/SILOptimizer/Utils/BasicBlockOptUtils.cpp

```diff
@@ -367,3 +367,12 @@ StaticInitCloner::clone(SingleValueInstruction *initVal) {
   }
   return cast<SingleValueInstruction>(getMappedValue(initVal));
 }
+
+// START: Workaround for C++ interop
```

Comment:
Did you pick a random file or is this the one I randomly picked? Would probably be better to put this in its own file, but given it's only needed for 5.9 ... 🤔